Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -38,11 +38,14 @@
 #include <AzToolsFramework/ViewportSelection/EditorSelectionUtil.h>
 #include <Atom/RPI.Public/RPISystemInterface.h>
 #include <AzCore/Settings/SettingsRegistryMergeUtils.h>
-#include <AzFramework/Entity/EntityDebugDisplayBus.h>
 
 #include <QMenu>
 #include <QAction>
 
+#ifdef _WIN32
+#include <tlhelp32.h>
```

Comment:
This needs to be PAL-fied.

Reply:
Agreed, it does, but this is a decent temp stop-gap for Windows..